geode-assembly/src/integrationTest/resources/assembly_content.txt

```diff
@@ -943,6 +945,7 @@ lib/lucene-analyzers-phonetic-6.6.2.jar
 lib/lucene-core-6.6.2.jar
 lib/lucene-queries-6.6.2.jar
 lib/lucene-queryparser-6.6.2.jar
+lib/micrometer-core-1.1.2.jar
```

Comment:
This should be 1.1.3 (which may or may not change the versions of HdrHistogram and LatencyUtils).